command/commands/token.py

```diff
@@ -18,7 +21,9 @@ class TokenCommand:
     success_msg = "This is your token:\n```\n{}\n```" \
                   "\nKeep it secret! Keep it safe!\nIt will expire at {}."
 
-    def __init__(self, db_facade, config):
+    def __init__(self,
+                 db_facade: DBFacade,
+                 config: 'TokenCommandConfig') -> None:
```

Comment:
Yes. It's for forward references. For some reason, just running mypy with some default settings allows you to take away the quotes (because it's referencing its own class). But when you run pytest with mypy plugin, it doesn't recognize its own class. Weird.